gallery/src/Application.tsx

```diff
@@ -4,92 +4,82 @@
-import NxViewportSizedExample from './styles/NxViewportSized/NxViewportSizedExample';
-import NxViewportSizedExpandingExample
-  from './styles/NxViewportSized/NxViewportSizedExpandingExample';
-import SectionScrollingWrapper from './styles/NxViewportSized/SectionScrollingWrapper';
-
-const pageMappings: PageMapping = mergeAll(values(pageConfig));
-
-function Page({ match, location }: RouteChildrenProps<{ pageName: string }>) {
-  const pageName = match ? match.params.pageName : null,
-      pageHeader = pageName || 'Home',
-      Content = pageName ? pageMappings[pageName] : Home;
-
-  useEffect(function() {
-    handleQueryParams(queryString.parse(location.search));
-  }, [location.search]);
-
-  if (Content) {
-    // Put a key on <main> so that it re-renders entirely on route change, resetting scroll position
-    return (
-      <main key={pageName || 'home'} className="nx-page-main">
-        <div className="nx-page-title">
-          <h1 className="nx-h1">
-            {pageHeader}
-          </h1>
-        </div>
-        <Content/>
-      </main>
-    );
-  }
-  else {
-    // unknown page
-    return <Redirect to="/" />;
-  }
-}
+import React from 'react';
+import {
+  faAtom,
+  faBiohazard,
+  faBolt,
+  faCrow,
+  faPlaceOfWorship
+} from '@fortawesome/free-solid-svg-icons';
+import { NxStatefulSidebarNavigation } from '@sonatype/react-shared-components';
 
 function Application() {
+  const navLinks = [
+    {
+      name: 'Link Name Alpha',
+      href: '/alpha',
+      icon: faAtom,
+      current: true
+    }, {
+      name: 'Link Name Beta',
+      href: '/beta',
+      icon: faBiohazard
+    }, {
+      name: 'Link Name Gamma',
+      href: '/gamma',
+      icon: faBolt
+    }, {
+      name: 'Link Name Delta',
+      href: '/delta',
+      icon: faCrow
+    }, {
+      name: 'Link Name Epsilon',
+      href: '/epsilon',
+      icon: faPlaceOfWorship
+    }, {
+      name: 'Link Name Zeta',
+      href: '/zeta',
+      icon: faAtom
+    }, {
+      name: 'Link Name Etha',
+      href: '/etha',
+      icon: faPlaceOfWorship
+    }, {
+      name: 'Link Name Theta',
+      href: '/theta',
+      icon: faBiohazard
+    }
+  ];
+  const attributions = [
+    'Powered by Nexux IQ Server',
+    'Created by Sonatype'
+  ];
+  const logoImg = require('./resources/logos/SON_hexagon_cropped.svg');
+
   return (
-    <Router>
-      <PageHeader />
+    <>
+      <NxStatefulSidebarNavigation isDefaultOpen={true}
+                                   logoImg={logoImg}
+                                   logoText="nexus lifecycle"
+                                   logoLink="/home"
+                                   links={navLinks}
+                                   helpLink="http://google.com"
+                                   helpText="google help"
+                                   collapsedReleaseText="release 105"
+                                   expandedReleaseText="lifecycle release 105"
```

Comment:
I wonder if the difference between these two should be worked out automatically by the component

Reply:
> I wonder if the difference between these two should be worked out automatically by the component

That's an assumption based on the instance of the mock, but it may well be a possibility that the two texts are entirely different.

Reply:
> but it may well be a possibility that the two texts are entirely different.

I think it would be up to design to say whether or not that's true.  Given that we've only got the one example so far, I can only guess.

Reply:
> > but it may well be a possibility that the two texts are entirely different.
> 
> I think it would be up to design to say whether or not that's true. Given that we've only got the one example so far, I can only guess.

@stephenim